src/Sarif/Sarif.csproj

```diff
@@ -24,7 +24,15 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
   <ItemGroup>
+    <Compile Remove="Baseline\**" />
     <Compile Remove="NotYetAutoGenerated\**" />
+    <EmbeddedResource Remove="Baseline\**" />
+    <None Remove="Baseline\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="Visitors\SarifCurrentToVersionOneVisitor.cs" />
```

Comment:
>SarifCurrentToVersionOneVisitor [](start = 30, length = 31)

Temporarily removed so I can isolate the v1/v2 transformation into a discrete change

Reply:
Ok.

---
In reply to: [244507727](https://github.com/Microsoft/sarif-sdk/pull/1191#discussion_r244507727) [](ancestors = 244507727)